ui/pages/snap-account-redirect/components/snap-account-redirect-context.tsx

```diff
@@ -8,7 +8,7 @@ import {
   ButtonVariant,
   Text,
 } from '../../../components/component-library';
-import { SnapAccountRedirectProps } from '../snap-account-redirect';
+import { type SnapAccountRedirectProps } from '../snap-account-redirect';
```

Comment:
might be more explicit to use `type { SnapAccountRedirectProps }` as it is in `ui/pages/snap-account-redirect/components/snap-account-redirect-message.tsx`